cmd/kubeadm/app/phases/addons/proxy/manifests.go

```diff
@@ -48,6 +48,40 @@ data:
         tokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
 `
 
+	// KubeProxyConfigMap is the proxy ConfigMap manifest
+	KubeProxyConfigMap19 = `
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: kube-proxy
+  namespace: kube-system
+  labels:
+    app: kube-proxy
+data:
+  kubeconfig.conf: |-
+    apiVersion: v1
+    kind: Config
+    clusters:
+    - cluster:
+        certificate-authority: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+        server: {{ .MasterEndpoint }}
+      name: default
+    contexts:
+    - context:
+        cluster: default
+        namespace: default
+        user: default
+      name: default
+    current-context: default
+    users:
+    - name: default
+      user:
+        tokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+  config.conf: |-
+    apiVersion: kubeproxyconfig/v1alpha1
+    kind: KubeProxyConfiguration
+{{ .ProxyConfig}}
+`
 	// KubeProxyDaemonSet is the proxy DaemonSet manifest
 	KubeProxyDaemonSet = `
```

Comment:
You need to duplicate this manifest in ...18 and ...19 as well; keep 18 as is and do this change in ...19 + remove `ExtraParams` and `ClusterCIDR` in 19

Reply:
done